test/cppsuite/src/main/database_operation.cpp

```diff
@@ -124,6 +124,20 @@ database_operation::populate(
     logger::log_msg(LOG_INFO, "Populate: finished.");
 }
 
+void
+database_operation::background_compact_operation(thread_worker *tc)
+{
+    logger::log_msg(
+      LOG_ERROR, type_string(tc->type) + " thread {" + std::to_string(tc->id) + "} commencing.");
```

Comment:
typo: `LOG_ERROR` -> `LOG_INFO`

Reply:
Nice catch! Fixed in b86f4b1d83a83009e3a9db555da0826fa88e811e